postgres/init/20-portal.sql

```diff
@@ -29,15 +29,21 @@ CREATE TABLE portal.company_roles (
 );
 
 
+CREATE TABLE portal.iam_clients (
+    id uuid PRIMARY KEY,
+    client_client_id character varying(255) NOT NULL,
+    CONSTRAINT uk_iam_client_client_client_id UNIQUE(client_client_id)
+);
+
 --
 -- Name: company_user_roles; Type: TABLE; Schema: public; Owner: -
 --
 
 CREATE TABLE portal.company_user_roles (
     id uuid PRIMARY KEY,
     company_user_role character varying(255) NOT NULL,
-    namede character varying(255) NOT NULL,
-    nameen character varying(255) NOT NULL
+    iam_client_id uuid NOT NULL,
+    CONSTRAINT fk_company_user_role_client_id FOREIGN KEY (iam_client_id) REFERENCES portal.iam_clients(id)
 );
```

Comment:
I would actually propose that we rename that section to "user_roles"
company_user_role is miss-leading for standalone roles

Reply:
ok, I'll rename the tables (and dependant tables)

Reply:
rename done